app/services/staging-adjustments-mapper.js

```diff
@@ -66,3 +67,48 @@ module.exports = function (workloadStagingIdStart, workloadStagingIdEnd, workloa
     })
   })
 }
+
+module.exports.mapGsAdjustments = function (workloadStagingIdStart, workloadStagingIdEnd, workloadReportId) {
+  return getWorkloadOwnersInWorkloadRange(workloadStagingIdStart, workloadStagingIdEnd, workloadReportId)
+  .then(function (workloadOwnerIds) {
+    return getStagingGs()
+    .then(function (stagingGsAdjustments) {
+      var gsAdjustments = []
+      var promises = []
+
+      if (stagingGsAdjustments) {
+        stagingGsAdjustments.forEach(function (gsAdjustment) {
+          promises.push(
+            getAdjustmentReasonFromCode(gsAdjustment.contactCode)
+            .then(function (adjustmentReason) {
+              return getWorkloadOwnerId(gsAdjustment.omKey, gsAdjustment.omTeamKey)
+              .then(function (workloadOwnerId) {
+                var startDate = new Date(gsAdjustment.contactDate)
+                var endDate = new Date(startDate)
+                endDate.setDate(endDate.getDate() + 30)
```

Comment:
Another potential for the `dateHelper`?

Reply:
End date is dependent on start date, so would require a function in date-helper, rather than just passing back test data. Worth creating a date-helper class in app rather test for things like this?

Reply:
Sorry, I misunderstood what was going on. That's fair enough, sounds like this is the cleanest solution.